src/suite/index.ts

```diff
@@ -141,6 +141,7 @@ function deployCron(
     nameSuffix,
     schedule,
     concurrencyPolicy = 'Forbid',
+    activeDeadlineSeconds = 300,
```

Comment:
Can you please remove the default? I'm afraid it might break existing cron jobs

Reply:
do we have jobs running more than 5 minutes?

Reply:
I dunno but I don't want to check it in production 😅